nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -864,6 +864,10 @@ CREATE ROLE postgres LOGIN SUPERUSER;
      Default algorithm for ZRAM swap was changed to <literal>zstd</literal>.
    </para>
    </listitem>
+   <listitem>
+    The installer now enables sshd by default. This improves installation on headless machines especially ARM single-board-computer.
+    To be able to login either a password or ssh key must be set for root or the nixos user.
```

Comment:
```suggestion
    To login through ssh, either a password or an ssh key must be set for the root user or the nixos user.
```

Reduce the risks of miscommunication, the local console still has auto-login. Additionally (maybe?) make the phrasing less disjointed?